test/e2e/features/step-definitions/bookings-list.js

```diff
@@ -0,0 +1,40 @@
+import { defineSupportCode } from 'cucumber'
+import { By, until } from 'selenium-webdriver'
+import { driver, url } from '../support/hooks'
+import addWeeks from 'date-fns/add_weeks'
+
+defineSupportCode(({ Given, When, Then }) => {
+  Given('I create a booking for next week', async () => {
+    await driver.get(`${url}/book`)
+    const start = addWeeks(new Date(), 1)
+    const end = new Date(start)
+    end.setMinutes(start.getMinutes() + 1)
+    const startForForm = start.toISOString().split('.')[0]
+    const endForForm = end.toISOString().split('.')[0]
+    const subjectInput = await driver.findElement(By.name('subject'))
+    await subjectInput.sendKeys('My Bookable for Next Week')
+    const startInput = await driver.findElement(By.name('start'))
+    await driver.wait(async () => (await startInput.getAttribute('value')).includes('T'))
+    await startInput.clear()
+    await startInput.sendKeys(startForForm)
+    const endInput = await driver.findElement(By.name('end'))
+    await driver.wait(async () => (await endInput.getAttribute('value')).includes('T'))
+    await endInput.clear()
+    await endInput.sendKeys(endForForm)
+    const createButton = await driver.findElement(By.tagName('button'))
+    await createButton.click()
```

Comment:
might want to refactor this as there's a lot of duplication with `Given I create a booking`